communication/src/main/java/PNCInterface.java

```diff
@@ -0,0 +1,31 @@
+import org.jboss.da.communcation.pnc.entity.BuildConfiguration;
+import org.jboss.da.communcation.pnc.entity.BuildConfigurationSet;
+import org.jboss.da.communcation.pnc.entity.Product;
+import org.jboss.da.communcation.pnc.entity.Project;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import java.util.List;
+
+public interface PNCInterface {
```

Comment:
@slaskawi Do you mean the `Configuration` file in the NCL codebase?
If so, do you think I should also adopt the "@class" and "@module-config" JSON format?